bin/browser.js

```diff
@@ -23,6 +23,11 @@ const getOutput = async (page, request) => {
         return output;
     }
 
+    if (request.action == 'cookie') {
```

Comment:
Use `===` here.

Reply:
makes definitely sense ... ;-)
Added Explanation to Readme, updated Changelog and corected browser.js

Hope I got all ;-)